src/blockdata/script.rs

```diff
@@ -41,7 +41,29 @@ use util::key::PublicKey;
 
 #[derive(Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash)]
 /// A Bitcoin script
-pub struct Script(Box<[u8]>);
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+pub struct Script(
+    #[cfg_attr(feature = "schemars", schemars(schema_with = "schemas::script"))]
+    Box<[u8]>);
+
+#[cfg(feature = "schemars")]
+mod schemas {
```

Comment:
Perhaps it makes sense to have an special module for all the schemas. Like `json_schemas.rs` and then refer with `schema_with = "::json_schemas::script"`? I think that'd be cleaner.